src/AbstractPlayerBridge.ts

```diff
@@ -1,11 +1,30 @@
-import type { Orientation, StreamPhase } from './InteractiveBridge';
 import { LiveryBridge } from './LiveryBridge';
+import type {
+  Config,
+  DisplayMode,
+  Features,
+  Orientation,
+  PlaybackDetails,
+  PlaybackMode,
+  PlaybackState,
+  Qualities,
+  StreamPhase,
+  UserFeedback,
+} from './util/schema';
+import {
+  displayMode,
+  userFeedback,
+  validateBoolean,
+  validateNumber,
+} from './util/schema';
 
 /**
  * Abstract player bridge class which implements part of the player side API based on browser logic
  * and defines abstract methods to be implemented to complete support for all InteractiveBridge commands.
  */
 export abstract class AbstractPlayerBridge extends LiveryBridge {
+  protected config?: Config;
```

Comment:
Note: I set config this way without creating a subscription here because I ran into an issue with triggering subscribeConfig here before this.player is set in the player-web PlayerBridge. So the subscribeConfig has to be set in that scope instead.